.packit.yaml

```diff
@@ -13,6 +13,8 @@ files_to_sync:
 upstream_package_name: polkit
 # downstream (Fedora) RPM package name
 downstream_package_name: polkit
+# arguments for OpenScanHub
```

Comment:
You may want to mention why is it enable here. For example, it only works at smaller scale as of now and can not be enabled for larger projects, but improves quality of GCC analyzer reports.

Reply:
I had this originally only in the commit message, but I agree that having it in the configuration file should be better for future traceability. Updated.